todoman/interactive.py

```diff
@@ -0,0 +1,583 @@
+import urwid
+from .ui import TodoEditor, EditState
+from .model import Todo
+
+import logging
+logging.basicConfig()
+
+logger = logging.getLogger()
+
+class TodomanItem(urwid.CheckBox):
```

Comment:
`TodoItem` might be more descriptive.

As for other classes, prefixing `Todoman` to their name sounds redundant (especially since they're already in package `todoman.interactive`.